@here/harp-mapview/lib/DataSource.ts

```diff
@@ -244,6 +244,15 @@ export abstract class DataSource extends THREE.EventDispatcher {
         // to be overloaded by subclasses
     }
 
+    /**
+     * Used to configure the ppi used by the `DataSource`.
+     *
+     * @param ppi Pixel-per-inch.
+     */
+    setPPI(ppi?: number): void {
```

Comment:
why not using a setter/getter property.

Reply:
To keep consistency with ``setLanguages``, since it's a similar decoder configuration option.